### PR TITLE
Test suite error detection enhancement

### PR DESCRIPTION
**Pull request by Dineshbabu-Ravichandran**
- Made changes to detect the errors in image HIP and HOST test suites and print them at the end

### CODE REVIEW

File: utilities/test_suite/common.py [hunk 1]
```diff
@@ -388,3 +391,71 @@ def dataframe_to_markdown(df):
...
+       result += "_PLN1_toPLN1"
+    return result
+
+def get_bit_depth(bitDepth):
+    result = str(bitDepthDict[bitDepth])
+    return result
+
+def get_signal_name_from_return_code(return_code):
+    result = ""
+    if return_code < 0:
```
Comment (HazarathKumarM) [inline utilities/test_suite/common.py:449]: please use camel case for all  variables

**Review by HazarathKumarM — COMMENTED** [review]
@Dineshbabu-Ravichandran  please resolve the comments

File: utilities/test_suite/HOST/runTests.py [hunk 2]
```diff
@@ -273,9 +291,12 @@ def rpp_test_suite_parser_and_validator():
         if case == "82" and (("--input_path1" not in sys.argv and "--input_path2" not in sys.argv) or qaMode == 1):
             srcPath1 = ricapInFilePath
             srcPath2 = ricapInFilePath
-        if case == "26" and (("--input_path1" not in sys.argv and "--input_path2" not in sys.argv) or qaMode == 1):
+        elif case == "26" and (("--input_path1" not in sys.argv and "--input_path2" not in sys.argv) or qaMode == 1):
             srcPath1 = lensCorrectionInFilePath
             srcPath2 = lensCorrectionInFilePath
+        else :
```
Comment (r-abishek) [inline utilities/test_suite/HOST/runTests.py:297]: No space before colon. Same on L333

File: utilities/test_suite/HOST/runTests.py [hunk 3]
```diff
@@ -72,8 +73,12 @@ def run_unit_test(srcPath1, srcPath2, dstPathTemp, case, numRuns, testType, layo
...
                     print("./Tensor_host " + srcPath1 + " " + srcPath2 + " " + dstPathTemp + " " + str(bitDepth) + " " + str(outputFormatToggle) + " " + str(case) + " " + str(noiseType) + " 0")
                     result = subprocess.Popen([buildFolderPath + "/build/Tensor_host", srcPath1, srcPath2, dstPathTemp, str(bitDepth), str(outputFormatToggle), str(case), str(noiseType), str(numRuns), str(testType), str(layout), "0", str(qaMode), str(decoderType), str(batchSize)] + roiList + [scriptPath], stdout=subprocess.PIPE, stderr=subprocess.PIPE)    # nosec
-                    stdout_data, stderr_data = result.communicate()
-                    print(stdout_data.decode())
+                    stdoutData, stderrData = result.communicate()
+                    print(stdoutData.decode())
+                    exitCode = result.returncode
+                    if(exitCode != 0):
+                        log_detected_errors("Returned non-zero exit status : "+ str(exitCode) + " " + stderrData.decode(), errorLog, imageAugmentationMap[int(case)][0], get_bit_depth(int(bitDepth)), get_image_layout_type(layout, outputFormatToggle, "HOST"), get_signal_name_from_return_code(exitCode))
```
Comment (r-abishek) [inline utilities/test_suite/HOST/runTests.py:80]: L76-L80 is repeating almost 36 times in this PR with minor changes in arguments, can we consolidate to put it in one function in common.py?

**Review by r-abishek — CHANGES_REQUESTED** [review]
@Dineshbabu-Ravichandran Pls address comments

**Review by r-abishek — APPROVED** [review]
@Dineshbabu-Ravichandran @HazarathKumarM  Hope all HIP/HOST tests are passing for this PR.

**Pull request merged**